### PR TITLE
Create the minimum viable product for project-1

### PR DESCRIPTION
**Pull request by Leoism**
Meet the current minimal requirements for project-1 which consist of a brief description of myself, some of my interests, previous projects, and brief history on me. In addition, all files were moved into `step/` to help with organization.

### CODE REVIEW

File: leoism/step/portfolio/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst [hunk 1]
```diff
@@ -0,0 +1 @@
+com/google/sps/servlets/DataServlet.class
```
Comment (andrewnguyen) [inline leoism/step/portfolio/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst]: Are files under target/ supposed to be checked in?

**Comment by Leoism** [comment]
Also, it looks like the original commit regarding to changes within `portfolio` disappeared when moving contents. I am not sure if that is okay or not

**Review by andrewnguyen — COMMENTED** [review]
Please add to .gitignore exclusions for files in a target/ directory. I think those are per-build specific files and should not be checked in.

**Comment by Leoism** [comment]
Okay, I added the .gitignore files for the target/ directory and places I think don't need to be checked in, such as walkthroughs.

**Comment by andrewnguyen** [comment]
Alright one last thing- can you amend the commit email to use your google account?

**Comment by andrewnguyen** [comment]
Actually take it or leave it: can you split up moving the files into the step/ folder and the parts where you make changes (ie where you write your bio) into two separate PRs? Sorry I should have asked this earlier

**Comment by Leoism** [comment]
I updated the email. running `git log` shows that the previous commits were all signed with the @google email. I am still trying to figure out properly splitting the PR but I will see what I can do

**Comment by Leoism** [comment]
I am closing this PR. I have split this PR into two, as suggested. They can be found at https://github.com/googleinterns/step155-2020/pull/10 and https://github.com/googleinterns/step155-2020/pull/11

**Pull request closed without merging**